charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,66 @@
+
+# Metadata Schema
+
+## Description
+
+The Human Cell Atlas (HCA) Metadata Schema are JSON format schema. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure [the FAIRness of the HCA data](https://www.nature.com/articles/sdata201618).
```

Comment:
Although JSON is currently used for the metadata and I am in no way asking for changes, I wonder if JSON should be specifically named in the charter. It seems too much implementation detail in the description and locks you into that format (if you ever decide to update).

Reply:
As JSON is so fundamental to how the schemas work I think it needs to be in the charter. If we were to decide to move to YAML or protobuff it would need rechartering so not mentioning JSON seems like needless obfuscation